packages/desktop-gui/src/project/project-model.js

```diff
@@ -231,6 +233,10 @@ export default class Project {
     return _.pick(this, 'id', 'path')
   }
 
+  isBrowserState (...browserStates) {
+    return _.includes(browserStates, this.browserState)
```

Comment:
Do we typically favor this approach (functional) or built-ins (`Array.prototype.includes`)?